### PR TITLE
Issue 1 create an entry -Euichang Park

### PR DESCRIPTION
**Pull request by EuichangPark**
Capture

### CODE REVIEW

**Comment by EuichangPark** [comment]
377width
780width
daty entry
entryId increases
photo preview

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+function handleSubmit(event) {
+  event.preventDefault();
+  var myObject = {};
+  var $title = document.querySelector('#title').value;
+  var $url = document.querySelector('#photo-url').value;
+  var $notes = document.querySelector('#notes').value;
+  myObject.title = $title;
+  myObject.url = $url;
+  myObject.notes = $notes;
+  myObject.entryId = data.nextEntryId + 1;
```
Comment (Zhiwutian) [inline js/main.js:23]: here you should just use `data.nextEntryId` then increment it on the next line as you are doing

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small functionality update needed

**Comment by EuichangPark** [comment]
377width
780width
entryId increases
input focus
photo preview

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**